moped-editor/src/views/projects/newProjectView/NewProjectView.js

```diff
@@ -153,6 +153,7 @@ const NewProjectView = () => {
           moped_proj_phases: {
             data: [
               {
+                phase_name: "potential",
```

Comment:
was this creating another bug?

Reply:
yes, that's why i had to redeploy

Reply:
from what I understand, when you create a new project in the new project view without this default value, the default `phase_name` is null and it causes an error.

Reply:
a separate bug from https://github.com/cityofaustin/atd-data-tech/issues/9403? Or related?

Reply:
same bug. it didn't work until i added this line